Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM opensciencegrid/software-base:release
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+RUN yum update -y && \
+    yum clean all && \
+    rm -rf /var/cache/yum/*
+
+RUN yum install -y oidc-agent && \
+    yum clean all && \
+    rm -rf /var/cache/yum/*
```

Comment:
Could you combine these into a single `RUN` step? The fewer layers in a container image, the better